### ISSUE TITLE
[IOS][Google Map] showDirections is not showing the route screen 

### ISSUE THREAD

**Issue by HasanAlqaisi** [mattermoran/map_launcher#188]
Hello,
When using `showDirections` with waypoints:
```dart
      await MapLauncher.showDirections(
        mapType: map,
        destination: Coords(
          coords.latitude,
          coords.longitude,
        ),
        waypoints: waypoints
                ?.mapIndexed(
                  (index, item) => Waypoint(
                    item.latitude,
                    item.longitude,
                    'Location ${index + 1}',
                  ),
                )
                .toList() ??
            [],
      );
```

The route is not showing directly on IOS, instead this screen will be showing:
<img width="377" alt="Screenshot 2024-08-29 at 1 19 11 PM" src="https://github.com/user-attachments/assets/9adf0091-9ae8-4fd8-9f5f-481a891ac285">

However, on Android it will show the directions screen directly:
<img width="337" alt="Screenshot 2024-08-29 at 1 22 10 PM" src="https://github.com/user-attachments/assets/9ea52e59-317c-4247-aa3f-f2c3f0c1c62e">

I am not sure why it behaves different on IOS.

**Comment by mattermoran** [comment]
it's just how it works on google maps for ios. nothing I can do about it. 

let me know if you think there's a way to show something else, otherwise I'm closing this for now

**Comment by HasanAlqaisi** [comment]
> it's just how it works on google maps for ios. nothing I can do about it.
> 
> let me know if you think there's a way to show something else, otherwise I'm closing this for now

I've found a way by using a different url. But I am not sure if it fits all the cases.
Here's a [fork](https://github.com/HasanAlqaisi/map_launcher) if you would like to check.
I couldn't find comprehensive information on this topic in the Google Maps documentation, so I found it through a search at that time.

**Comment by mattermoran** [comment]
hmm interesting. so this one works as expected you say? I would prefer not to use undocumented apis as that could just stop working at any point without notice but def something to investigate. will reopen for now

**Comment by HasanAlqaisi** [comment]
> hmm interesting. so this one works as expected you say? I would prefer not to use undocumented apis as that could just stop working at any point without notice but def something to investigate. will reopen for now

Yeah I've used it in a project and it was showing the map the same as Android.
But I haven't tested it enough, also maybe it is documented somewhere because I came across this method through multiple Stack Overflow questions.

**Comment by Stcstcdpidpi** [comment]
Good

**Comment by mattermoran** [comment]
I have just tested on latest version and it seems to be working correctly on ios now. 
let me know if you still experiencing issues

<img width="295" height="639" alt="Image" src="https://github.com/user-attachments/assets/333534c9-123b-4d1c-9e22-02d6bad698bf" />
